weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/ReactionsLoaded.tsx

```diff
@@ -273,3 +276,10 @@ export const ReactionsLoaded = ({
     </Tailwind>
   );
 };
+
+function feedbackIsLegacy(feedbackType: string) {
+  return (
+    feedbackType.startsWith('wandb.note') ||
+    feedbackType.startsWith('wandb.reaction')
```

Comment:
Why are these legacy? I still see emoji reactions and notes persisted with those feedback types.